.semaphore/pre-release.yml

```diff
@@ -22,8 +22,18 @@ blocks:
       jobs:
         - name: "Publish to NPM"
           commands:
+            - git checkout -b pre-release-$SEMAPHORE_PIPELINE_ID
+            - git push origin pre-release-$SEMAPHORE_PIPELINE_ID --no-verify
             - nvm use
             - npx lerna bootstrap
             - yarn
-            - yarn lerna version prerelease --yes
+            - yarn lerna version --conventional-commits --conventional-prerelease --no-changelog --no-push --yes
             - yarn lerna publish from-package --yes --dist-tag next
+            - git add packages/*.md packages/*/index.d.ts
+            - git commit -m 'chore(Publish): automate readme, type definitions and changelog update' --no-verify
+            - git push origin pre-release-$SEMAPHORE_PIPELINE_ID --follow-tags --no-verify
```

Comment:
Commit the changes for readme and declaration files. These tasks were done by the `postpublish` script before. But since now I want to do the publishing work on a clean branch, so I have to explicitly say push to xx branch rather than just master branch like before.
Also I removed `postpublish` script in `package.json`